mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -95,8 +95,11 @@ final class EMLDAOptimizer extends LDAOptimizer {
    * Compute bipartite term/doc graph.
    */
   override private[clustering] def initialize(docs: RDD[(Long, Vector)], lda: LDA): LDAOptimizer = {
+    val docConcentration = breeze.stats.mean(lda.getDocConcentration.toBreeze)
```

Comment:
To check for exact equality, I would check every element, rather than checking the mean (which could be slightly off due to numerical inaccuracy).